src/elements/DeployedList/DeployedList.wc.svelte

```diff
@@ -141,18 +141,22 @@
   function _createK8sRows(rows: any[]) {
     return rows.map((row, i) => {
       const { name, master, workers, consumption } = row;
-      const publicIp = master.publicIP ?? ({} as any);
-      return [i + 1, name, publicIp.ip || "None", publicIp.ip6 || "None", master.planetary || "None", workers, consumption]; // prettier-ignore
+      let publicIp = master.publicIP ?? ({} as any);
+      master.publicIP? publicIp.ip = publicIp.ip.replace(/.{0,3}$/, '') : publicIp.ip = "None";
```

Comment:
that's will work for 2 digits subnets. but if it's one digit, it will take one more digit from the ip itself.
IMO, a better solution is to split the ip `publicIp.ip.split("/")[0]`, this will work even if the ip is not containing the subnet part

Reply:
done.